src/openvr/openvr_data.h

```diff
@@ -136,12 +137,12 @@ class openvr_data {
 	struct texture_material {
 		TextureType type;
 		vr::TextureID_t texture_id;
-		Ref<SpatialMaterial> material;
+		Ref<StandardMaterial3D> material;
```

Comment:
BaseMaterial3d?

Reply:
I remember looking into the source / changelogs in git and coming to the conclusion that Spatial became Standard.

I probably checked other bits of the engine and compared 3.2 to 4.0 to see what was done during the refactor.

I don't remember exactly; but I'm pretty confident it was right :)

Reply:
Whatever of the superclass of StandardMaterial3D is.

Reply:
Yup, I think you're right there. BaseMaterial is the underlying class and is used in meshes because you can either assign a shader material or spatial, now standard, material. But this code creates a standard material because we're just using the build in shader.

Reply:
```
StandardMaterial3D

Inherits: BaseMaterial3D < Material < Resource < Reference < Object
```

ORMMaterial3D is the other child of BaseMaterial3D.